tests/ballerina-compiler-api-test/src/test/java/io/ballerina/semantic/api/test/SymbolBIRTest.java

```diff
@@ -296,4 +297,29 @@ public int hashCode() {
             return Objects.hash(this.name, this.kind);
         }
     }
+
+    @DataProvider(name = "hasDefaultTestProvider")
+    public Object[][] hasDefaultTest() {
+        return new Object[][]{
+                {21, 6, null},
+                {22, 6, true},
+                {23, 6, false}
+        };
+    }
+
+    @Test(dataProvider = "hasDefaultTestProvider")
+    public void testHasDefaultValue(int line, int col, Object hasDefault) {
+
```

Comment:
Extra line here as well

Reply:
Removed in 56b1bcd76d10af1ed0cd6ce23d365c5e9326506d